src/Angor/Client/wwwroot/index.html

```diff
@@ -29,18 +29,33 @@
     <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
     <meta http-equiv="Pragma" content="no-cache">
     <meta http-equiv="Expires" content="0">
+ 
 </head>
 
 <body>
     <div id="app">
         <div class="loader-wrapper">
             <div class="loader"></div>
+            <script>
+                (function() {
+                    const theme = localStorage.getItem('theme');
+                    const loader = document.querySelector('.loader-wrapper');
+                    if (theme === 'dark') {
```

Comment:
This is good way to check and set theme, not using the local storage variable directly because that can be modified by attackers.